R/appStatic.R

```diff
@@ -84,26 +84,35 @@ messageShiny <- function() {
   )
 }
 copyLogos <- function(logo, directory) {
+  # Create 'www' directory if it doesn't exist
   dir.create(paste0(directory, "/www"), showWarnings = FALSE)
-  from <- system.file("/www/images/hds_logo.svg", package = "omopViewer")
-  to <- paste0(directory, "/www/hds_logo.svg")
-  file.copy(from = from, to = to, overwrite = TRUE)
-  if (!is.null(logo)) {
-    if (logo == "HDS") {
-      return("hds_logo.svg")
-    } else if (file.exists(logo)) {
-      nm <- basename(logo)
-      to <- paste0(directory, "/www/", nm)
-      file.copy(from = logo, to = to, overwrite = TRUE)
-      return(nm)
-    } else {
-      cli::cli_warn(c("!" = "Logo couldn't be found."))
-      return(NULL)
-    }
+  hdsLogo <- system.file("/www/images/hds_logo.svg", package = "omopViewer")
+  ohdsiLogo <- system.file("/www/images/ohdsi_logo.svg", package = "omopViewer")
+  if (is.null(logo)) {
```

Comment:
logo = NULL is an option as this is the way to give the user the chance to not add a logo, so I would allow that (no warning)